### PR TITLE
fix duplicates phantoms in the auth panel

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**

### CODE REVIEW

File: auth/auth_user.py [hunk 1]
```diff
@@ -9,20 +9,19 @@
 
 
 class AuthUser:
-	phantom_set = None
 	user_profile = None # Cache the user
-
+	phantom_ids = []
```
Comment (mark-at-pieces) [inline auth/auth_user.py:13]: lets adjust this to be a dictionary, unless order here is required, if order does not matter then lets us a dictionary

File: auth/auth_user.py [hunk 2]
```diff
@@ -9,20 +9,19 @@
...
-		if cls.phantom_set:
-			return cls.phantom_set
-		cls.phantom_set = sublime.PhantomSet(PiecesSettings.output_panel, "pieces_auth")
-		return cls.phantom_set
+	def create_new_phantom(cls,html):
+		if cls.phantom_ids:
+			for phantom in cls.phantom_ids:
+				PiecesSettings.output_panel.erase_phantom_by_id(phantom)
+		cls.phantom_ids.append(PiecesSettings.output_panel.add_phantom("auth_phantom", 
+			sublime.Region(0, 0), html, sublime.LAYOUT_INLINE))
```
Comment (mark-at-pieces) [inline auth/auth_user.py:21]: we need to remove our previous panel everytime? and the previous panels before?

**Review by mark-at-pieces — CHANGES_REQUESTED** [review]
1. lets remember to prioritize use of dictionaries instead of arrays,(unless order matters)

2. why do we need to iterate over our previous panels and remove them

File: auth/auth_user.py (hunk 2 above)
Comment (Bishoy-at-pieces) [reply auth/auth_user.py]: yes we need to clear it.

**Review by mark-at-pieces — APPROVED** [review]

**Pull request merged**